postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -159,6 +182,40 @@ def _dbtags(self, db, *extra_tags):
             t.extend(self._tags_no_db)
         return t
 
+    def _get_active_connections(self):
+        start_time = time.time()
+        extra_query_config = ""
+        params = ()
+        if self._config.dbstrict:
```

Comment:
We could reuse the construction of this query filter between the two pg_stat_activity queries.

Reply:
ah yeah I meant to DRY that up yesterday, thanks for the reminder